title.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>

```

Comment:
either add a button link to index.html or add directly to index.html surrounding the canvas